isthmus/build.gradle.kts

```diff
@@ -99,7 +99,7 @@ dependencies {
 }
 
 graal {
-  mainClass("io.substrait.isthmus.PlanEntryPoint")
```

Comment:
Any specific reason for name refactoring?

Reply:
> Any specific reason for name refactoring?

Main reason is that initially this class offered support for Plan, now it is needed to offert support for Plan and Expression. The initial name could be maintained, but the scope of the class should be documented very clearly.